docs/start/index.rst

```diff
@@ -81,6 +89,25 @@ If you have installed with git, pull the QCoDeS repository using your
 favourite method (git bash, git shell, github desktop, ...). There are
 new commits to the repository daily.
 
+Keeping your environment up to date
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Additional dependencies are periodically added to the QCoDeS environment and
+new versions of packages that QCoDeS depends on are released.
+
+To keep the QCoDeS environment upto date you can run
+
+.. code:: bash
+
+   conda env update
+
+from the root of your QCoDeS git repository. Alternatively you can grab a new
+version of the ``environment.yml`` file from above and run the same command from a
```

Comment:
what if we add a reference to `conda env update` command, so that the users can read how to use this command themselves instead of us explaining all the details?

Reply:
Sure but the documentation that I can find is rather bad https://conda.io/docs/commands/env/conda-env-update.html do you have a better link

Reply:
no, i don't)) it's not bad, it's just not formatted properly. i hope they fix it someday...